Ihr2 3081 (#253)
dotwebstack/dotwebstack-framework: service/openapi/README.md

breweries(name: String): [Brewery!]!
```

> All `Query` and `Path` parameters provided in a request should exist in the schema. If this not the case for a given request,

Reviewer: GraphQL or OAS schema?

> the application will return a `400` error status and a message stating which of the given parameters are not allowed.  

Reviewer: ```suggestion
the application will return a `400` response with a message stating which of the given parameters is not allowed.  
```
